### PR TITLE
Add Toast notify snackbar css

### PR DESCRIPTION
**Pull request by rahulcs754**
add Toast/Notify/Snackbar css 
image

Please review and thanks in advance

### CODE REVIEW

**Comment by iamsaurav28** [comment]
give  title name snackbar to ur html instead of document,  snackbar at bottom side color has relatable color as ur body color try to give different color so it can get highlight or else its okay everything looks good

File: css/components/toast_notify_snackbar.css [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+    bottom: 1rem;  
+    right: 3rem;
+}
+
+/*
+|| Notify
+*/
+
+.notify{
+    display: flex;
```
Comment (vaibhav-mahalle) [inline css/components/toast_notify_snackbar.css:37]: you can try to reuse the display flex. It is repeating

File: css/components/toast_notify_snackbar.css [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+*/
+ 
+.snackbar-text1{
+    color:hsla(0,0%,100%,.87);
+    padding: 0rem 0.2rem;
+    margin : 0.2rem 0rem;
+}
+
+.snackbar-bottom{
+    display: flex;
```
Comment (vaibhav-mahalle) [inline css/components/toast_notify_snackbar.css:159]: same as above

**Review by vaibhav-mahalle — COMMENTED** [review]
everything else looks good.

File: css/components/toast_notify_snackbar.css (hunk 1 above)
Comment (rahulcs754) [reply css/components/toast_notify_snackbar.css:37]: like as flex class

File: css/components/toast_notify_snackbar.css (hunk 2 above)
Comment (rahulcs754) [reply css/components/toast_notify_snackbar.css:159]: like a flex class i have already but i want to only show  snackbar class

File: css/components/toast_notify_snackbar.css [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+    flex-direction: row;
+    justify-content: flex-end;
+    min-width: 100%;
+    padding: 0rem 0.2rem;
+    margin:1rem 0rem;
+}
+
+.snackbar-retry1{
+    font-weight: 400;
+    color:#bb86fc;
```
Comment (shubhamkumar648) [inline css/components/toast_notify_snackbar.css]: use varaible of color

**Review by shubhamkumar648 — COMMENTED** [review]
conside my comment to change ! everything looks good

**Review by shubhamkumar648 — APPROVED** [review]
Good to merge

**Review by vaibhav-mahalle — COMMENTED** [review]
looks good. no changes required.

**Pull request merged**